src/com/projectkorra/projectkorra/event/PlayerBindChangeEvent.java

```diff
@@ -1,78 +1,84 @@
 package com.projectkorra.projectkorra.event;
 
 import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
 import org.bukkit.event.Event;
 import org.bukkit.event.HandlerList;
 
 /**
  * Called when a player binds or unbinds an ability
- *
- * @author savior67
  */
-public class PlayerBindChangeEvent extends Event {
+public class PlayerBindChangeEvent extends Event implements Cancellable {
 
-	private static final HandlerList handlers = new HandlerList();
+	private static final HandlerList HANDLERS = new HandlerList();
+	
+	private boolean cancelled = false;
+	
 	private final Player player;
 	private final String ability;
-	private final int slot; // slot is -1 if it is a multiability.
-	private final boolean isBinding; // true if the ability is being binded, otherwise false.
-	private final boolean isMultiAbility; // true if the ability is a multiability.
-	private boolean cancelled;
+	private final int slot; 
+	private final boolean isBinding, isMultiAbility; 
 
-	// bind event for abilities.
-	public PlayerBindChangeEvent(final Player player, final String ability, final int slot, final boolean isBinding) {
+	public PlayerBindChangeEvent(Player player, String ability, int slot, boolean isBinding, boolean isMultiAbility) {
 		this.player = player;
 		this.ability = ability;
 		this.slot = slot;
 		this.isBinding = isBinding;
-		this.cancelled = false;
-		this.isMultiAbility = false;
+		this.isMultiAbility = isMultiAbility;
 	}
 
-	// used for multi abilities.
-	public PlayerBindChangeEvent(final Player player, final String ability, final boolean isBinding) {
-		this.player = player;
-		this.ability = ability;
-		this.slot = -1;
-		this.isBinding = isBinding;
-		this.cancelled = false;
-		this.isMultiAbility = true;
+	public Player getPlayer() {
+		return this.player;
 	}
 
 	public String getAbility() {
 		return this.ability;
 	}
 
-	@Override
-	public HandlerList getHandlers() {
-		return handlers;
-	}
-
-	public static HandlerList getHandlerList() {
-		return handlers;
-	}
-
-	public Player getPlayer() {
-		return this.player;
-	}
-
+	/**
+	 * Get the slot being changed. 
+	 * <ul>
+	 * <li>In the case of binding a multiability, returns 0
+	 * <li>In the case of unbinding a multiability, returns the original slot
```

Comment:
I feel like you should expand upon this and make it so the documentation reflects that this is for normal ability bind changes as well.

Reply:
I agree